pkg/config/config.go

```diff
@@ -266,6 +266,17 @@ type EngineConfig struct {
 	// in containers-registries.conf(5).
 	CompatAPIEnforceDockerHub bool `toml:"compat_api_enforce_docker_hub,omitempty"`
 
+	// ComposeProviders specifies one or more external providers for the
+	// compose command.  The first found provider is used for execution.
+	// Can be an absolute and relative path or a (file) name.  Make sure to
+	// expand the return items via `os.ExpandEnv`.
```

Comment:
Can we add a builtin function
ComposeProvidersExpanded()

Which does the os.ExpandEnv for the caller?

Reply:
I considered it but there is only one caller which will call `os.ExpandEnv` on demand, so I thought it's OK.